### DISCUSSION TITLE
Is this dependency conflict warning concerning?

### DISCUSSION THREAD

**Discussion by fusion4github** [trpc/trpc#3066] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/67457575/197445561-49e55b3c-531d-439f-be10-54ad4daf7389.png)

Hey guys, I'm kinda new to TCRP stuffs as well as NextJS but my first experience with TCRP is fab. I really liked it. Kudos to the creator as well as the maintainers. 
I tried to deploy my side project which uses TCRP to vercel and got the above warning.
I'm using the following versions of TCRP:

![image](https://user-images.githubusercontent.com/67457575/197445761-cc48d981-2ca6-4e71-8bcb-3f646541c8f9.png)
(I assume they are all latest and `@trpc/react` dep is still on beta.20 version.

Any help/advice would be helpful. Thank You!

**Comment by sachinraja** [comment, marked as answer]
See the migration guide: https://trpc.io/docs/v10/migrate-from-v9-to-v10#renaming-of-trpcreact-to-trpcreact-query

`@trpc/react` was renamed to `@trpc/react-query` so you should remove it as a dependency now

**Reply by fusion4github** [reply]
Thanks :)
